src/test/java/org/sonar/plugins/stash/issue/collector/SonarQubeCollectorTest.java

```diff
@@ -142,4 +142,24 @@ public void testExtractIssueReportWithOldOption() {
     assertTrue(report.countIssues("severity1") == 0);
     assertTrue(report.countIssues("severity2") == 1);
   }
+  
+  @Test
+  public void testExtractIssueReportWithNoInputFile(){

```

Comment:
The test name is not relevant enough to reflect what it is really verified here.
We are checking that the process carry on even if an inputFile is null.